tools/bench/bench.md

```diff
@@ -71,8 +71,8 @@ Each test is defined as a profile is parameterized with a load specification.  S
 | --- | --- | --- |
 | Startup/Shutdown Time | Offline | 36% improvement |
 | Startup/Shutdown Time | Online | 23% improvement |
-| Parallel performance logging scalars | Offline | 83% improvement |
-| Parallel performance logging scales | Online | 88% improvement |
```

Comment:
i think scalars is an important info (can add it in parenthesis)

Reply:
Updated